docs/reference/overview.md

```diff
@@ -7,10 +7,20 @@ slug: /reference/
 
 This section contains general reference material for Camunda Cloud, including API references.
 
+## APIs
 - [Console API clients](cloud-console-api-clients.md)
 - [Console API (REST)](cloud-console-api-reference.md)
 - [Zeebe API (gRPC)](grpc.md)
 - [Tasklist API (GraphQL)](tasklist-api/generated.md)
+
+## Security, support, & license information
+- [Announcements](announcements.md)
+- [Licenses](licenses.md)
+- [Notices](notices.md)
```

Comment:
Similar to my comment on the notices page, in the heading we are listing "Security", but then non of the sub-pages contains the word security and I'm not sure which page to visit. Please ignore again if I'm just not aware that "notices" is commonly understood term for security reporting.

Reply:
Took care of this, "Notices" will now be "Security Notices".